pkg/gds/models/v1/pagination.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+)
+
+// Load a PageCursor from a page_token string.
+// This is very similar code to trtl.internal but decoupled so Trtl and GDS aren't
```

Comment:
I agree with you on decoupling GDS and TRTL, though I think future us may find it convenient to have a unified concept of pagination throughout the project.

Reply:
I'm certainly open to exploring it.